data/json/items/tools.json

```diff
@@ -6193,6 +6193,22 @@
     }
   },
   {
+    "id": "hand_drill",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "hand drill",
+    "description": "A primitive manual drill with a single drill bit.  It is slow and it will exhaust you quickly.",
```

Comment:
Needs low chance to spawn in `tools_carpentry`, say perhaps weighting `20`. Note the entries are alphabetical.